cmd/prober/prober.go

```diff
@@ -170,8 +176,11 @@ func main() {
 	verificationCounter.With(prometheus.Labels{verifiedLabel: "false"}).Add(0)
 	verificationCounter.With(prometheus.Labels{verifiedLabel: "true"}).Add(0)
 
-	go runProbers(ctx, frequency, oneTime)
-
+	if fulcioClient, err := NewFulcioClient(); err != nil {
+		Logger.Errorf("error creating fulcio grpc client %v", err)
```

Comment:
Should this be `Fatalf` since if the client isn't properly initialized, it will fail again later?

Reply:
It will not fail again. I agree that should be a Fatalf instead Error